mantidproject/mantid
# Save Additional Metadata to SANS Reduced Output

**Description of work**
Added back some metadata that had gone missing in the reduction output since Mantid 3.9.2.

**Purpose of work**
Retain metadata in the output needed for analysis down the road.

Requested by user @smk78 

**Summary of work**
- Adjusted SANSSave to make use of all of the saving parameters in SaveCanSAS1D
- Added those same properties to SaveNXCanSAS and use them in SANSSave
- Added loading for that metadata to LoadNXCanSAS

**To test:**

_Note: You may need to download a new version of the Training Course Data if you get errors about `Version 0` when loading the user file._

1. Open the SANS GUI
2. Load the user file and the batch file from the `loqdemo` from the Training Course Data
3. Save output to file (make sure your default mantid save directory is somewhere sensible)
5. Process All
6. Check your save directory and find the output file. Make sure the output is in a similar state to the before example in the issue. 

Fixes #28998 


---

#### Reviewer ####

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

#### Gatekeeper ####

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.
